src/components/ApplicationContext/ApplicationContext.tsx

```diff
@@ -127,14 +158,58 @@ export const ApplicationProvider: React.FC = () => {
       .catch(() => setStatus(AppStatus.Errored))
   }, [])
 
+  const saveData = () => {
+    const { fname, lname, phone, email } = contactFormData
+    const {
+      age,
+      pronouns,
+      race,
+      sexuality,
+      school,
+      collegeAffiliation,
+      eventLocation,
+      major,
+      currentStanding,
+      country,
+    } = demographicFormData
+    const { whyCruzHacks, newThisYear, grandestInvention } = shortAnswerFormData
+    const { firstCruzHacks, hackathonCount, priorExperience } =
+      priorExperienceFormData
+    const { linkedin, github, cruzCoins, anythingElse } = connectedFormData
+    const { conductAgree, tosAgree } = mlhFormData
+
+    const data: SavedApplication = {
+      contact: { fname, lname, phone, email },
+      demographic: {
+        age,
+        pronouns,
+        race,
+        sexuality,
+        school,
+        collegeAffiliation,
+        eventLocation,
+        major,
+        currentStanding,
+        country,
+      },
+      shortAnswer: { whyCruzHacks, newThisYear, grandestInvention },
+      priorExperience: { firstCruzHacks, hackathonCount, priorExperience },
+      connected: { linkedin, github, cruzCoins, anythingElse },
+      MLH: { conductAgree, tosAgree },
+    }
+    store("application", data, 60 * 60 * 24 * 30, user?.email)
+  }
+
   const nextPage = () => {
     if (page < 6) {
+      saveData()
       setPage(page + 1)
     }
   }
```

Comment:
It might be good to maybe save the farthest page visited so we can use that in progress bar on page 0 for a started application. We should probably save this if design thinks it's good and in a separate PR.

Reply:
Oh yeah, that would be cool. There are still a few other things that I want to address before we merge this (unless we need this by Sunday?).

Reply:
I would like this by Sunday, but if there are things that need to be addressed we should prioritize that.